src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+} from '@material-ui/core';
+import { TodoProps } from './typings';
+
+const Todo: FC<TodoProps> = ({id, todo, onRemove, onToggle}: TodoProps) => {
+  const { title, done } = todo;
+  const statusLabel = done ? 'Reativar' : 'Completar';
+  const opacity = done ? '0.5' : 1;
+
+  return (
+    <Card      
+      data-testid={`todo-${id}`}
+      key={id}
+      variant="outlined"
+      sx={{
+        width: '300px',
```

Comment:
Material will change to px automatically.
```suggestion
        width: 300,
```

Reply:
I copied and pasted